tests/python_tests/ov_genai_test_utils.py

```diff
@@ -0,0 +1,219 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import pathlib
+import os
+import pytest
+import functools
+import openvino
+import openvino_tokenizers
+import openvino_genai as ov_genai
+from typing import Union, List, Dict, Tuple, Optional
+from pathlib import Path
+import shutil
+import json
+
+
+def get_models_list():
+    precommit_models = [
+        "katuni4ka/tiny-random-phi3",
+    ]
+
+    nightly_models = [
+        "TinyLlama/TinyLlama-1.1B-Chat-v1.0",
+        "facebook/opt-125m",
+        "microsoft/phi-1_5",
+        "microsoft/phi-2",
+        "THUDM/chatglm2-6b",
+        "Qwen/Qwen2-0.5B-Instruct",
+        "Qwen/Qwen-7B-Chat",
+        "Qwen/Qwen1.5-7B-Chat",
+        "argilla/notus-7b-v1",
+        "HuggingFaceH4/zephyr-7b-beta",
+        "ikala/redpajama-3b-chat",
+        "mistralai/Mistral-7B-v0.1",
+        
+        # "meta-llama/Llama-2-7b-chat-hf",  # Cannot be downloaded without access token
+        # "google/gemma-2b-it",  # Cannot be downloaded without access token.
+        # "google/gemma-7b-it",  # Cannot be downloaded without access token.
+        "meta-llama/Llama-2-13b-chat-hf",
+        "meta-llama/Meta-Llama-3-8B-Instruct",
+        "openlm-research/open_llama_3b",
+        "openlm-research/open_llama_3b_v2",
+        "openlm-research/open_llama_7b",
+        "databricks/dolly-v2-12b",
+        "databricks/dolly-v2-3b",
+    ]
+
+    if pytest.run_marker == "precommit":
+        model_ids = precommit_models
+    else:
+        model_ids = nightly_models
+
+    prefix = pathlib.Path(os.getenv('GENAI_MODELS_PATH_PREFIX', ''))
+    return [(model_id, prefix / model_id.split('/')[1]) for model_id in model_ids]
+
+
+def get_chat_models_list():
+    precommit_models = [
+        "Qwen/Qwen2-0.5B-Instruct",
+    ]
+
+    nightly_models = [
+        "TinyLlama/TinyLlama-1.1B-Chat-v1.0",
+        "meta-llama/Meta-Llama-3-8B-Instruct",
+        "meta-llama/Llama-2-7b-chat-hf",
+        # "google/gemma-2b-it",  # Cannot be downloaded without access token
+        # "google/gemma-7b-it",  # Cannot be downloaded without access token
+    ]
+
+    if pytest.run_marker == "precommit":
```

Comment:
`python tests/python_tests/test_chat_generate_api.py -m precommit` results in
```
Traceback (most recent call last):
  File "C:\Users\vzlobin\r\g\tests\python_tests\test_chat_generate_api.py", line 33, in <module>
    @pytest.mark.parametrize("model_descr", get_chat_models_list())
                                            ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\vzlobin\r\g\tests\python_tests\ov_genai_test_utils.py", line 70, in get_chat_models_list
    if pytest.run_marker == "precommit":
       ^^^^^^^^^^^^^^^^^
AttributeError: module 'pytest' has no attribute 'run_marker'
```

How do I run it?

Reply:
It should've worked. For some reason `run_marker` was not set. 
https://github.com/pavel-esir/openvino.genai/blob/2d90fe1ea981c29d181a8f6894040b73b458448e/tests/python_tests/conftest.py#L17-L19

It works on my machine and seems to work on pre-commit
![image](https://github.com/openvinotoolkit/openvino.genai/assets/5703039/80a40282-f53b-45ce-a48a-fcc26f2a1645)